react-js/react-components-and-hierarchies.md

```diff
@@ -34,36 +35,31 @@
 - [How to pass data to props.children](https://frontarm.com/james-k-nelson/passing-data-props-children/)
 - [React Strap](https://reactstrap.github.io/) 
 
-
-<br/>
-
 ### Lesson
 
-#### What is the difference between Components and built-in JSX tags like <button>
+#### What is the difference between Components and built-in JSX tags like button ?
 
-Difference between Components and built-in JSX tags are :
-- Components are created by the user while JSX tags are pre defined
-- Components are usually self closing in nature (eg: ```<App/>, <Header/>```) while tags are in pairs(```<button>this is a tag</button>, <p>this is a paragraph tag </p>```)
+The difference between Components and built-in JSX tags is that components are created by the user while JSX tags are pre-defined
```

Comment:
I would also add that pre-defined JSX mirror the html elements, while components would not. Linking to React's full list would help.

Reply:
As mentioned, please link to React's list of tags: https://zhenyong.github.io/react/docs/tags-and-attributes.html#html-elements